src/OrleansAzureUtils/Hosting/AzureSilo.cs

```diff
@@ -61,13 +71,43 @@ public AzureSilo()
             logger = LogManager.GetLogger("OrleansAzureSilo", LoggerType.Runtime);
         }
 
+        public async Task<bool> ValidateConfiguration(ClusterConfiguration config)
+        {
+            if (config.Globals.LivenessType == GlobalConfiguration.LivenessProviderType.AzureTable)
+            {
+                string deploymentId = serviceRuntimeWrapper.DeploymentId;
+                string connectionString = serviceRuntimeWrapper.GetConfigurationSettingValue(DataConnectionConfigurationSettingName);
```

Comment:
Only get values from the runtime if these fields are not set in the configuration